server/cp/ingest/parser/cp_onclusive.py

```diff
@@ -88,6 +88,19 @@ def parse(self, content, provider=None):
                 # remove duplicates
                 item["anpa_category"] = unique(category)
                 item["subject"] = unique(item["subject"])
+
+                # update event status SDCP-749
+                if item.get("is_provisional", False):
+                    eocstat_map = get_resource_service("vocabularies").find_one(
```

Comment:
imo fetching it for every item is not very efficient, would be needed just once

Reply:
and you can use `get_items` method which will only return active ones so you don't have to filter it further

Reply:
all right !